ui/app/ducks/swaps/swaps.js

```diff
@@ -39,7 +39,6 @@ import { calcGasTotal } from '../../pages/send/send.utils'
 import {
   decimalToHex,
   getValueFromWeiHex,
-  hexMax,
```

Comment:
Nit: `hexMax` is now unused, so that function can be removed here as well.

Reply:
I'm always hesitant to remove useful utility functions but that promotes rot, so good shout.